smac/facade/epils_facade.py

```diff
@@ -0,0 +1,418 @@
+import logging
```

Comment:
I assume a lot of code here is copied from the SMAC-facade. Maybe we should have a file utils.py in the facade directory which contains helper functions to for example retrieve the TAE in order to avoid code duplications.